pkg/client/request.go

```diff
@@ -82,7 +82,7 @@ var mapEncodingTypeToEncoding = map[BodyEncodingType]BodyEncoding{
 type JSONBodyEncoding struct{}
 
 func (j *JSONBodyEncoding) ContentType() string {
-	return fmt.Sprintf("application/vnd.puppet.nebula.%v+json", API_VERSION)
+	return fmt.Sprintf("application/vnd.puppet.relay.%v+json", API_VERSION)
```

Comment:
Are you OK with holding on this PR until we can get the relevant update into the API to match?

Reply:
sure though FWIW i used the binary built with this to talk to the service and it worked fine 🤷

Reply:
i'm going to merge and will just hold on cutting a new release